doc/commentary.txt

```diff
@@ -6,6 +6,9 @@ License: Same terms as Vim itself (see |license|)
 Comment stuff out.  Then uncomment it later.  Relies on 'commentstring' to be
 correctly set, or uses b:commentary_format if it is set.
 
+Set b:commentary_startofline to true to insert comment characters into
```

Comment:
Rather than raising the question of "what is true", say `Assign `b:commentary_startofline`.

Reply:
Ok.